apps/condo/domains/organization/schema/plugins/addOrganizationFieldPlugin.js

```diff
@@ -0,0 +1,40 @@
+const { plugin } = require('@core/keystone/plugins/utils/typing')
+const { Relationship } = require('@keystonejs/fields')
+const get = require('lodash/get')
+const { getById } = require('@core/keystone/schema')
+
+const addOrganizationFieldPlugin = ({ fromField }) => plugin(({ fields = {}, ...rest }) => {
+    fields['organization'] = {
+        schemaDoc: 'Ref to the organization. It is filled in on the server and is read-only',
+        type: Relationship,
+        ref: 'Organization',
+        isRequired: true,
+        knexOptions: { isNotNullable: true }, // Relationship only!
+        kmigratorOptions: { null: false, on_delete: 'models.CASCADE' },
```

Comment:
If organization will be -> object will be deleted too. Do we want to delete MeterReadings from mobile app for example?

Reply:
Unfortunately you. Because the meter reading is useless without the organization